src/dataframe/dataframe.jl

```diff
@@ -102,6 +103,14 @@ function DataFrame(; kwargs...)
     return result
 end
 
+function DataFrame(pairs::Pair{Symbol}...)
```

Comment:
Yes we could. It would look something like:
```julia
function DataFrame(; kwargs...)
    function pair(t) 
        k, v = t
        k => v
    end
    DataFrame(map(pair, kwargs)...)
end
```
Look good?